tooling/components/io.siddhi.distribution.editor.core/src/main/java/io/siddhi/distribution/editor/core/internal/ExportUtils.java

```diff
@@ -153,16 +153,30 @@ public File createZipFile() throws DockerGenerationException, KubernetesGenerati
         boolean buildDocker = false;
         if (exportType != null && exportType.equals(EXPORT_TYPE_DOCKER)) {
             if (exportAppsRequest.getDockerConfiguration() != null) {
-                if (exportAppsRequest.getDockerConfiguration().getImageName() != null &&
-                        !exportAppsRequest.getDockerConfiguration().getImageName().equals("")) {
-                    String dockerImageName = exportAppsRequest
-                            .getDockerConfiguration()
-                            .getImageName()
-                            .replaceAll("/", "-")
-                            .replaceAll(":", "-").trim();
-                    zipFileName = dockerImageName + ".zip";
-                    zipFileRoot = dockerImageName + File.separator;
+                if (exportAppsRequest.getDockerConfiguration().isPushDocker()) {
+                    if (exportAppsRequest.getDockerConfiguration().getImageName() != null &&
```

Comment:
As discussed offline, let's use the same config in both cases, IMO it simplifies the logic

Reply:
Added the changes in here https://github.com/siddhi-io/distribution/pull/773/commits/77ad1a7fdc21c270c11b2ebcc8e8c76cbd69dd26